composer.json

```diff
@@ -12,10 +12,10 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
```

Comment:
I tried that but got a weird bug with composer in the tests: 
https://travis-ci.org/ruflin/Elastica/builds/47757241